utils/data/target_metadata/lowz_galaxy_db_metadata.csv

```diff
@@ -1,3 +1,3 @@
-target_name_ullyses,target_name_simbad,simbad_link,target_classification,gaia_id,targ_ra,targ_dec,coordinate_epoch,host_galaxy_name,host_cluster_name,spectral_type,spectral_type_ref,sp_class,temp_class,lum_class,u_mag,u_mag_ref,b_mag,b_mag_ref,v_mag,v_mag_ref,bmv0_mag,bmv0_mag_ref,extinction_ebmv,extinction_ebmv_ref,star_teff,star_teff_ref,surface_gravity,surface_gravity_ref,star_luminosity,star_luminosity_ref,star_radius,star_radius_ref,star_mass,star_mass_ref,rotation_velocity,rotation_velocity_ref,terminal_velocity,terminal_velocity_ref,rate_massloss,rate_massloss_ref,known_binary,known_binary_ref,F225W_STmag,F275W_STmag,F336W_STmag,F475W_STmag,F814W_STmag
-NGC3109,NGC3109,https://simbad.u-strasbg.fr/simbad/sim-id?Ident=ngc3109,Galaxy,,150.7786542,26.15957222,2000,NGC3109,None,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Sextans-A,Sex A,https://simbad.u-strasbg.fr/simbad/sim-id?Ident=sextans+a,Galaxy,,152.7520833,-4.69166667,2000,Sextans-A,None,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
\ No newline at end of file
+target_name_ullyses,target_name_simbad,simbad_link,target_classification,gaia_id,targ_ra,targ_dec,coordinate_epoch,host_galaxy_name,host_cluster_name,spectral_type,spectral_type_ref,sp_class,temp_class,lum_class,u_mag,u_mag_ref,b_mag,b_mag_ref,v_mag,v_mag_ref,bmv0_mag,bmv0_mag_ref,extinction_ebmv,extinction_ebmv_ref,star_teff,star_teff_ref,surface_gravity,surface_gravity_ref,star_luminosity,star_luminosity_ref,star_radius,star_radius_ref,star_mass,star_mass_ref,rotation_velocity,rotation_velocity_ref,terminal_velocity,terminal_velocity_ref,rate_massloss,rate_massloss_ref,known_binary,known_binary_ref,F225W_STmag,F275W_STmag,F336W_STmag,F475W_STmag,F814W_STmag
+NGC3109,NGC3109,https://simbad.u-strasbg.fr/simbad/sim-id?Ident=ngc3109,Galaxy,,150.7786542,26.15957222,2000,NGC 3109,None,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
Actually, I think this needs to stay as `NGC3109` because there is logic in the catalog front-end to check host galaxy name, and therefore that check won't evaluate correctly if there's a space.

Reply:
@jotaylor do I need to revert back Sextans A to Sextans-A, too then?

Reply:
Actually, I checked the code in the MAST frontend repo, and they do a wildcard search (`"Sextans%A,NGC%3109`) so I think both should be okay! Apologies for the false alarm